w3lib/url.py

```diff
@@ -34,7 +34,7 @@ def _quote_byte(error):
 
 _safe_chars = RFC3986_RESERVED + RFC3986_UNRESERVED + EXTRA_SAFE_CHARS + b'%'
 
-def safe_url_string(url, encoding='utf8', path_encoding='utf8'):
+def safe_url_string(url, encoding='utf8', quote_path=None, path_encoding='utf8'):
```

Comment:
I think boolean default value would be more clear for a boolean argument than ``None``. Also I think default value should be ``True`` in this case :)

Reply:
Also would be nice to document the new argument in the docstring below.